lib/api/bucketDeleteCors.js

```diff
@@ -33,7 +33,8 @@ function bucketDeleteCors(authInfo, request, log, callback) {
         }
         log.trace('found bucket in metadata');
 
-        if (!isBucketAuthorized(bucket, requestType, canonicalID, authInfo, log, request)) {
+        if (!isBucketAuthorized(bucket, requestType, canonicalID, authInfo, log, request,
+            request.actionImplicitDenies)) {
```

Comment:
Maybe a bit too late for this comment, but as we pass the request object to this function, maybe the function's last parameter could be removed to avoid passing it twice?

Reply:
PS: it's @williamlardier 😄